src/CourseUtil.h

```diff
@@ -83,6 +83,10 @@ class CourseID
 			return sPath < other.sPath;
 		return sFullTitle < other.sFullTitle;
 	}
+	bool operator==(const CourseID& rhs) const
```

Comment:
Any reason why != wasn't overloaded either? If one is overloaded, the other(s) should be derived from them.